board/flags-config.cmake

```diff
@@ -59,6 +59,10 @@ elseif(BOARD MATCHES "RPI4" OR BOARD MATCHES "CORTEXA72")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mtune=cortex-a72 -ftree-vectorize")
   set(OPENBLAS_TARGET "CORTEXA72")
   set(OPENBLAS_BINARY "64")
+elseif(BOARD MATCHES "RPI5")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=armv8.2-a+crypto+fp16+rcpc+dotprod -fasynchronous-unwind-tables")
+  set(OPENBLAS_TARGET "CORTEXA76")
+  set(OPENBLAS_BINARY "64")
```

Comment:
The problem with this is that, someone needs to put RPI5, and in the future, the best is to have the architecture, since these are generic more or less.
So it would be best to do like the above and change RPI5 to CORTEXA72.
But also we need to change line 64, So openblas builds correctly

Reply:
I agree.  I do wonder if there are some cases where the `-mtune=` will not be everything we can do, but we can figure that out later.  I removed the RPI5 definition in 0c2ff48. :+1: